cmd/compgen/cmd/receiver/receiver.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	common "github.com/liatrio/compgen/cmd/common"
+	"github.com/spf13/cobra"
+)
+
+var shortDescription = "Build a new Open Telemetry receiver component"
+var longDescription = `
+receiverName: A full module path (https://go.dev/ref/mod#glos-module-path)
+  E.g. 'github.com/liatrio/liatrio-otel-collector/receiver/myreceiver'
+
+outputDir: A full or relative path to a directory that contains receivers
+	E.g. receiver/`
+
+// ReceiverCmd represents the receiver command
+var ReceiverCmd = &cobra.Command{
+	Use:   "receiver [flags] receiverName outputDir",
+	Short: shortDescription,
+	Long:  fmt.Sprint(shortDescription, "\n", longDescription),
+	Args:  cobra.MinimumNArgs(2),
+	Run:   run,
+}
+
+//go:embed templates/*
+var Templates embed.FS
+
+func init() {
```

Comment:
If this is an empty function, do we need to keep it?

Reply:
That's boilerplate from cobra. I'm actually not sure if it _must_ exist. I'll try removing it and see what happens

Reply:
It probably doesn't need to exist if it's empty. but i'd add a help arg in there instead of removing it.

Reply:
Compgen works fine without it. Deleted. Which is awesome!